### ISSUE TITLE
1.7 timed out error

### ISSUE THREAD

**Issue by JoeSGT** [HexagonMC/BungeeCord#236]
### What Hexacord version are you using? Post the output of /bungee
I have tested on versions
v222
v198 - what cause the disconnection for 1.7.10
v181 - works without any issues

### Are you using Bungee plugins? If yes, please list all of them + versionnumber
i have a clean no plugins bungee during the testing

### How can we reproduce the error/bug? 
basically i have a modpack for 1.7 yes i know it's old but still i am having issues when i update to v181 to v222 and any 1.7.10 clients are unable to connect. the  pull request #157 from dogboy21/patch-1 i think it's not the issue because i had remember using this patch before you guys merge it so it's somewhere else but i can be wrong but players are unable to connect this they are getting "timed out" on their clients 

### Is an error message (bungee/spigot/client)? If yes, use pastebin.com and share the link.
this is what it shows for the Thermos MC 1.7 server but on the bungeecord side of it give no errors https://pastebin.com/raw/k7SprJJp

### In addidtion to that, please explain your issue as detailed as possible.
basically using up to date you are unable to use 1.7.10 instead use bungeecord v181

**Comment by i9hdkill** [comment]
Try latest build
